docs/REVIEWING.md

```diff
@@ -36,6 +36,40 @@ following items should also be checked:
 - [ ] Does it comply with [component spec](specs/component.md)?
 - [ ] Does it comply with the [instrumentation spec](specs/instrumentation.md)?
 
+
+### Sink review checklist
+
+This checklist is specific for Vector's sink code.
+
+## Logic
+
+- [ ] Does it work? Do you understand what it is supposed to be doing?
+- [ ] Does the retry logic make sense?
+- [ ] Are the tests testing that the sink is emitting the correct metrics?
+- [ ] Are there integration tests?
+
+## Code structure
+
+- [ ] Is it using the sink prelude?
+- [ ] Is it gated by sensible feature flags?
+- [ ] Is the code modularized into `mod.rs`, `config.rs`, `sink.rs`,  `request_builder.rs`, `service.rs`
```

Comment:
Maybe this is implicit but could be good to call out that it should be using the new style / service framework.

Reply:
Yeah, gosh. How did I miss that..?